frontend/src/views/web/challenge/submission.html

```diff
@@ -291,62 +291,64 @@ <h5 class="w-300">Make Submission</h5>
-                                                        <span><strong>{{attribute.name}}</strong> - {{attribute.description}} <span ng-if="attribute.required === true" class="color-orange">*</span></span><br>
-                                                        <input type="text" ng-model= "attribute.value" name="{{attribute.name}}" id="{{attribute.name}}" ng-required="attribute.required === true">
+                                                        <input type="text" ng-model="attribute.value" name="{{attribute.name}}" id="{{attribute.name}}" ng-required="attribute.required === true">
                                                         <span class="form-icon"><i class="fa fa-text"></i></span>
+                                                        <label for="{{attribute.name}}">{{attribute.name}} ({{attribute.description}}) <span ng-if="attribute.required === true" class="color-orange">*</span></span></label>
                                                     </div>
                                                 </div>
                                             </div>
                                             <div ng-if="attribute.type == 'radio'">
                                                 <div class="row">
-                                                    <div class="input-field align-left col s10">
-                                                        <span><strong>{{attribute.name}}</strong> - {{attribute.description}} <span ng-if="attribute.required === true" class="color-orange">*</span></span><br>
-                                                        <md-radio-group ng-model="attribute.value">
-                                                            <md-radio-button ng-repeat="option in attribute.options" ng-value="option" ng-model="attribute.value">
-                                                                <span class="fs-16">{{option}}</span>
-                                                            </md-radio-button>
-                                                        </md-radio-group>
+                                                    <div class="align-left col s10">
+                                                        <label class="label-radio" for="{{attribute.name}}">{{attribute.name}} ({{attribute.description}}) <span ng-if="attribute.required === true" class="color-orange">*</span></span></label>
+                                                        <div class="input-field">
+                                                            <md-radio-group ng-model="attribute.value" id="{{attribute.name}}" name="{{attribute.name}}">
+                                                                <md-radio-button ng-repeat="option in attribute.options" ng-value="option" ng-model="attribute.value">
+                                                                    <span class="fs-16">{{option}}</span>
```

Comment:
The font-size for the other options (answers not labels) is `0.9em` IIRC. Can you please make sure that this is same as that.

Reply:
It `1.0em` for other option element from create challenge page. Which is same as these options

Reply:
What about the other submission meta attributes, what is the value for them?

Reply:
It is the same

Reply:
perfect! Thanks for checking.